### ISSUE TITLE
License Missing

### ISSUE THREAD

**Issue by georg-eckert-zeiss** [MerjTek/MerjTek.WpfIntegration#1]

**Comment by georg-eckert-zeiss** [comment]
Hi. Could you please add a license file like MIT?

Thank you,
Georg

**Comment by georg-eckert-zeiss** [comment]
The link to the blog post is not working anymore. But I found it in the [Microsoft blog archives](https://docs.microsoft.com/en-us/archive/blogs/nicgrave/feed.xml).

Cheers,
Georg

**Comment by georg-eckert-zeiss** [comment]
If you want, I would happily put up a pull request with the suggested changes.

Cheers,
Georg

**Comment by georg-eckert-zeiss** [comment]
I reached out to Nick Gravelyn and he answered, the code can be considered MIT or Microsoft Public License. Whichever fits most. I also asked him to answer here if possible.

**Comment by nickgravelyn** [comment]
Yep either license is fine for my original code. I think most things I did at the time I put under MS-PL but I’m also good with MIT.

**Comment by georg-eckert-zeiss** [comment]
Thank you for your answer.
